sbndaq-artdaq-core/Overlays/FragmentType.hh

```diff
@@ -27,6 +27,7 @@ enum FragmentType : artdaq::Fragment::type_t {
   ICARUSTriggerUDP = artdaq::Fragment::FirstUserFragmentType + 10,
   ICARUSPMTGate = artdaq::Fragment::FirstUserFragmentType + 11,
   ICARUSTriggerV2 = artdaq::Fragment::FirstUserFragmentType + 15,
+  ICARUSTriggerV3 = artdaq::Fragment::FirstUserFragmentType + 17,
```

Comment:
why not 16?

Reply:
I see a TDCTIMESTAMP under SBND that is labeled as 16. I just chose the next available number as I assumed we would need in order to not cause major issues with previous data from both experiments.

Reply:
Ah, ok, I missed that 16. Then of course this is fine.